iroh-net/Cargo.toml

```diff
@@ -87,6 +87,8 @@ tracing-subscriber = { version = "0.3", features = ["env-filter"], optional = tr
 # metrics
 iroh-metrics = { version = "0.16.0", path = "../iroh-metrics", default-features = false }
 strum = { version = "0.26.2", features = ["derive"] }
+pin-project-lite = "0.2.14"
+base64 = "0.22.1"
```

Comment:
These are not for the metrics section.  maybe we should just remove the "sections" and simply keep one sorted list.

We also already have `pin_project` in the deps.  We should probably not have both, at least not directly, indirectly we get both anyway I think.  Personally I much prefer the attribute-style over this function-style as it integrates much better with tooling generally, no structs inside macros.  And the compilation cost isn't that much.